src/internal/components/selectable-item/styles.scss

```diff
@@ -64,6 +64,7 @@
   }
 
   &.selected {
+    z-index: 2;
```

Comment:
I don't think we should be playing with the z-index in dropdowns, why is this needed?